### PR TITLE
[WEBDEV-801] Labels

### PR DESCRIPTION
**Pull request by katylouise**
* Added the ability to process rdfs:label from n-triple data and create a Grom::Labels.
* Created a Grom::Response which returns Grom::Node objects and a Grom::Labels object.

### CODE REVIEW

File: lib/grom/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Grom
-  VERSION = '1.0.1'.freeze
+  VERSION = '1.1.1'.freeze
```
Comment (mattrayner) [inline lib/grom/version.rb]: This should be 1.1.0

File: spec/grom/reader_spec.rb [hunk 2]
```diff
@@ -14,6 +15,10 @@
       expect(subject.instance_variable_get(:@data)).to eq(data)
     end
 
+    it 'stores a Grom::Response object into @response' do
```
Comment (mattrayner) [inline spec/grom/reader_spec.rb:18]: Should there be another test for the labels method?

File: lib/grom/version.rb (hunk 1 above)
Comment (katylouise) [reply lib/grom/version.rb]: Whoops, of course!

File: spec/grom/reader_spec.rb (hunk 2 above)
Comment (katylouise) [reply spec/grom/reader_spec.rb:18]: Not sure what you mean here?  I have tested the populating of @labels_by_subject further down on line 100. Then there is also testing on the Grom::Response class to check the @labels method.

**Pull request closed without merging**